### PR TITLE
Release 4.1.0

### PR DESCRIPTION
**Pull request by jturel**

### CODE REVIEW

**Comment by ekohl** [comment]
I've applied the Bug label to it. This means it gets the slightly more specific Fixed bugs header. Other than that :+1:

**Comment by jturel** [comment]
ack. updated!

File: CHANGELOG.md [hunk 1]
```diff
@@ -274,7 +281,7 @@
 - Refs [\#5815](https://projects.theforeman.org/issues/5815) - generate certs for node qpid [\#14](https://github.com/theforeman/puppet-certs/pull/14) ([iNecas](https://github.com/iNecas))
 - fixes [\#5486](https://projects.theforeman.org/issues/5486)  prefix and candlepin url incorrect for rhsm template on dev... [\#13](https://github.com/theforeman/puppet-certs/pull/13) ([dustints](https://github.com/dustints))
 - Refs [\#5423](https://projects.theforeman.org/issues/5423) - fix certs generation for capsule usage [\#12](https://github.com/theforeman/puppet-certs/pull/12) ([iNecas](https://github.com/iNecas))
-- Fixing \#5299: variables not used properly. [\#11](https://github.com/theforeman/puppet-certs/pull/11) ([omaciel](https://github.com/omaciel))
+- Fixing [\#5299](https://projects.theforeman.org/issues/5299): variables not used properly. [\#11](https://github.com/theforeman/puppet-certs/pull/11) ([omaciel](https://github.com/omaciel))
```
Comment (ekohl) [inline CHANGELOG.md]: This is an odd one: it's not related to that redmine issue at all.

Comment (jturel) [reply CHANGELOG.md]: Hmm, I guess we should leave it consistently mismatched :)

Comment (ekohl) [reply CHANGELOG.md]: I was considering changing the PR title but let's just leave it as you have here.

**Comment by ehelms** [comment]
@jturel Since you opened this PR more changes have gone in which I think necessitates a regeneration of the changelog?

**Comment by jturel** [comment]
@ehelms updated!

**Comment by ehelms** [comment]
@jturel One more update please

**Comment by jturel** [comment]
updated

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,21 @@
 # Change Log
 
+## [4.0.2](https://github.com/theforeman/puppet-certs/tree/4.0.2) (2018-05-29)
```
Comment (ekohl) [inline CHANGELOG.md]: This might have been lost in a comment, but should we make this a 4.1.0 release?

**Comment by jturel** [comment]
@ekohl I think an update does make sense - made it so

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**